### ISSUE TITLE
Git LFS quota exceeded

### ISSUE THREAD

**Issue by lu2000luk** [EvanZhouDev/open-genmoji#2]
You tried uploading the whole model on Github. Unfortnunatley Git only supports files up to 100mb and to upload larger stuff you must use Git LFS. Github automatically does that but only gives you 1 GB of storage if you dont upgrade it. Now the model file is just a 143 bytes file containing the pointer info and the model is not downloadable/clonable. I whould suggest uploading the model to [HuggingFace](https://hf.co/) (for free) and then making a script to download it from there.

TLDR:
You can't upload files that bit on github for free
Upload them on [Huggingface](https://hf.co/) and make a script to download them in the repo.

**Comment by ItsAlphaNeon** [comment]
`Downloading lora/flux-dev.safetensors (209 MB)
Error downloading object: lora/flux-dev.safetensors (7b29f7c): Smudge error: Error downloading lora/flux-dev.safetensors (7b29f7cc636d582366ec0f2b4bfb4cfc3af94cd93fc0c7c27a16475d976b0d43): batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.`

If you could upload the model on Huggingface, we'd appreciate that. Thanks!

**Comment by EvanZhouDev** [comment]
Doing this right now. Sorry for the inconvenience!

**Comment by EvanZhouDev** [comment]
All done. Check updated documentation for download steps, as well as https://huggingface.co/EvanZhouDev/open-genmoji for the LoRA. Enjoy!

**Comment by lu2000luk** [comment]
Thanks!
